src/app/components/header/header.component.scss

```
@@ -0,0 +1,70 @@
+.header{
```

```suggestion
.header {
```

Have you added sass?

---

yes vaia. It was added while bootstrapping the project.